Building on Linux, gmake doesn't like ../contrib/fonts/wqy_1?pt.bdf as a target

### Describe the bug

Build completes up to the point of building the executable, but then gmake (4.4.1) complains:

```gmake[3]: *** No rule to make target '../contrib/fonts/wqy_1?pt.bdf', needed by 'all-am'.  Stop.```

which it is getting from res_DATA in src/Makefile.in, line 384.  Because all-am depends on ```$(DATA)```, which is set to ```$(res_DATA)```, make wants to build that target but can't.  Removing that line from the makefile fixes things for me, but I don't know if that's going to break anything else.

### Steps to reproduce the behaviour

Build current sources from git on a Linux box.  I'm building in a separate directory (obj/).

### Expected behavior

Build completes normally.

### What operating system(s) this bug have occurred on?

Linux aelfric 6.14.0-gentoo #1 SMP PREEMPT_DYNAMIC Mon Mar 31 04:46:11 EDT 2025 x86_64 Intel(R) Core(TM) i7-7700 CPU @ 3.60GHz GenuineIntel GNU/Linux

### What version(s) of DOSBox-X have this bug?

dcc08d7ece38f51b70ed1f279bfa8ac1142b0789

### Used configuration

```ini

```

### Output log

```text

```

### Additional information

_No response_

### Have you checked that no similar bug report(s) exist?

- [x] I have searched and didn't find any similar bug report.

### Code of Conduct & Contributing Guidelines

- [x] I agree to follow the code of conduct and the contributing guidelines.

--- a/src/Makefile.am
+++ b/src/Makefile.am
@@ -15,7 +15,8 @@ res_DATA = \
 	../dosbox-x.reference.conf \
 	../dosbox-x.reference.full.conf \
 	../contrib/fonts/FREECG98.BMP \
-	../contrib/fonts/wqy_1?pt.bdf \
+	../contrib/fonts/wqy_11pt.bdf \
+	../contrib/fonts/wqy_12pt.bdf \
 	../contrib/fonts/Nouveau_IBM.ttf \
 	../contrib/fonts/SarasaGothicFixed.ttf \
 	../CHANGELOG

